digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -31,6 +34,12 @@
 public class TDOperator
         implements Closeable
 {
+    private static final Logger logger = LoggerFactory.getLogger(TDOperator.class);
+
+    private static final int INITIAL_RETRY_WAIT = 100;
```

Comment:
Actually, as some operators still use the retry executor directly without the persistent retry that this PR implements, I'll revert this to the original 3 retries with 500ms initial wait for now. We can change this later when all operators have been state-machinified.